MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyROIPresenter.h

```diff
@@ -58,7 +58,6 @@ class ITomographyROIPresenter {
     ColorRangeUpdated,     ///< The uses changes the values or options of the
                            ///< color range
     SelectCoR,             ///< Start picking of the center of rotation
-    FindCoR,               ///< Run calculate CoR
```

Comment:
What happened to this and changes below? were they not meant to be part of this PR?